src/main/java/com/sourcery/gymapp/backend/workout/dto/ResponseRoutineDto.java

```diff
@@ -0,0 +1,13 @@
+package com.sourcery.gymapp.backend.workout.dto;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+public record ResponseRoutineDto(
+
+        UUID id,
+        String name,
+        String description,
+        LocalDateTime localDateTime
```

Comment:
naming should refer more to what it provides in Json.
I guess it's creation date, so creationDate?

Reply:
Tru didnt see that, gonna change to createdAt